app/models/user.js

```diff
@@ -33,12 +37,27 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.BOOLEAN,
         field: 'is_admin',
         defaultValue: false
+      },
+      points: {
+        type: DataTypes.INTEGER,
+        field: 'points',
+        defaultValue: 0
+      },
+      position: {
```

Comment:
This is good. However, you could also do a calculation before the final return (use the util i mean). But is OK for me.

Reply:
yes, the calculation will implement it in the next trello card